tardis/adapters/sites/cloudstack.py

```diff
@@ -94,29 +96,29 @@ def handle_exceptions(self):
         except asyncio.TimeoutError as te:
             raise TardisTimeout from te
         except ClientConnectionError:
-            logging.info("Connection reset error")
+            logger.warning("Connection reset error")
             raise TardisResourceStatusUpdateFailed
         except CloudStackClientException as ce:
             if ce.error_code == 535:
-                logging.info("Quota exceeded")
-                logging.debug(str(ce))
+                logger.warning("Quota exceeded")
+                logger.warning(str(ce))
```

Comment:
Here are also interesting changes of log level, `info` -> `warning` and `debug` -> `warning`. Especially the jump from debug to warning is interesting. As this is actually not two warnings, we might think about joining the strings?